app/src/main/java/se/inera/intyg/minaintyg/error/ErrorController.java

```diff
@@ -0,0 +1,38 @@
+package se.inera.intyg.minaintyg.error;
```

Comment:
Could we move this to the logging package instead of having a package called 'error'. This functionality makes sense if it belonged to logging.